src/cli/utils/flags.go

```diff
@@ -167,6 +177,36 @@ func AddSiteFlag(cmd *cobra.Command) {
 		"Backup data by site URL; accepts '"+Wildcard+"' to select all sites.")
 }
 
+func AddMaintenanceSafetyFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.StringVar(
+		&MaintenanceSafetyFV,
+		MaintenanceSafetyFN,
+		string(control.FullSafety),
+		"Disable safety measures to run maintenance faster. Improper usage can break a repository")
+	cobra.CheckErr(fs.MarkHidden(MaintenanceSafetyFN))
+}
+
+func AddQuickMaintenanceFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.BoolVar(
+		&QuickMaintenanceFV,
+		QuickMaintenanceFN,
+		false,
+		"Run quick maintenance that won't clean up and optimize the repo as much")
```

Comment:
```suggestion
		"Run a faster maintenance that does minimal clean-up and optimization")
```

Reply:
We can benefit from some of the Kopia docs here https://kopia.io/docs/advanced/maintenance/